mediaXsm.css

```diff
@@ -39,26 +47,87 @@
 
   .menu {
     list-style-type: none;
-    margin-right: 10%;
-    flex-direction: row;
+    position: absolute;
```

Comment:
- [OPTIONAL] Great job so far :raised_hands: but I think it would be better if you could use a `position: fixed;`  instead of absolute. Fixed position prevents your navigation page menu page from scrolling. The navigation is always fixed to the page instead of scrolling. Please change this. This will make your website similar to the original design. 💯 🥇